### DISCUSSION TITLE
Migrate from Cirris-CI to GitHub Actions

### DISCUSSION THREAD

**Discussion by bjlittle** [pletzer/mint#67] [Ideas]
Hey @pletzer

Glad to hear that the CI is working so well for you, but I wanted to float the idea of moving the CI from `cirrus-ci` to GitHub Actions (GHA).

What's attractive about GHA is that most of the Scientific Python community understand and use it, so there's a lesser barrier for newcomers to MINT. Plus I've found that their infrastructure spins up quicker than `cirrius-ci`, which is one of the downsides of `cirrus-ci` i.e., their free community tier can be oversubscribed, hence their business model is pushing you to paying for credits for lightning fast CI (which makes total sense).

Anyways, let me know what you think.

I'm happy to put in the effort to migrate the infrastructure across, but only if you're happy for me to do that.

Cheers

**Comment by pletzer** [comment]
Thanks @bjlittle . Absolutely happy with you moving to GHA. Thanks for your contributions @bjlittle
